openfe/setup/molecule.py

```diff
@@ -25,6 +25,14 @@ def rdkit(self) -> RDKitMol:
         """RDKit representation of this molecule"""
         return self._rdkit
 
+    @property
```

Comment:
when testing the network creation, it would be handy to probe Molecule a bit more than just hashes